### ISSUE TITLE
Permitir importação do CSV de candidatos pelo admin do django

### ISSUE THREAD

**Issue by ricardopoppi** [unidoscontraacorrupcao/ej-server#99]
Para manter as infos dos candidatos sempre atualizadas a importação será manual

**Comment by ricardopoppi** [comment]
@davidCarlos essa é a estrutura da planilha:

Estrutura da planilha (formato google drive): https://docs.google.com/spreadsheets/d/1F-y23v3YL0DF8kln0Ea-dPZ2MosP3Jk0tLJjjprwq9w/edit?usp=sharing


Aqui uma versão dela com os dados dos senadores que a gente já pode utilizar:
[exemplo_dados_candidatos.xlsx](https://github.com/unidoscontraacorrupcao/ej-server/files/2273600/exemplo_dados_candidatos.xlsx)

A diferença entre essas duas planilhas é que a primeira tem dois campos extras na parte do "pacto pela democracia": 
(i) Gravou video?
(ii) url video pacto
Peço ao @parrahenri indicar se esses campos vão existir mesmo na planilha, ou se foi uma sugestão que acabou sendo descartada

**Comment by parrahenri** [comment]
To tentando que não tenhamos!

**Comment by davidCarlos** [comment]
Realizei a importação dos dados presentes na planilha que o @ricardopoppi colocou no comentario. Seria interessante apressar o pessoal para enviar uma planilha completa, incluindo as fotos dos candidatos e outras informações necessárias para o app. Essa primeira planilha foi simples de ser importada, mas são esses campos faltantes que vão tornar mais complexo esse processo.

**Comment by parrahenri** [comment]
Já entrei nesse circuito das fotos! O que sabemos é:

TSE não disponibilizou ainda e vai fazer isso aos poucos, ou seja, só devemos ter fotos a partir do dia 15/8 e teremos de fazer cargas sucessivas.

Pedi pra nos anteciparem quais as dimensões das imagens e teremos de criar uma imagem default "foto ainda não foi disponibilizada pelo TSE" c/c: @luizapeixe

**Comment by parrahenri** [comment]
Sobre fotos:

As fotos, por exemplo, dos 11 candidatos que mandei ontem estão no
Divulgacand de 2010 ou 2014. Exemplo:

http://divulgacandcontas.tse.jus.br/divulga/#/candidato/2014/680/DF/70000001075

http://divulgacandcontas.tse.jus.br/divulga/#/candidato/2010/14417/MA/100000000206

O formato em 2018 é o mesmo. Claro que cada candidato atualiza a foto
de eleição para eleição

**Comment by luizapeixe** [comment]
subi agora uma sugestão de imagem default pros cards lá na issue #126 

sobre o formato das fotos, hj no TSE elas tem 166x229 px, né? Elas virão nesse formato pra nós? nosso layout prevê hoje a dimensão de 320x250 px. Precisamos pensar numa solução pra isso?

**Comment by parrahenri** [comment]
sim, não temos 100% de certeza, mas provavelmente virá nessas dimensões sim

**Comment by ricardopoppi** [comment]
@davidCarlos (cc @parrahenri ) segue a planilha para importação: [exemplo_dados_candidatos_versaocomlinksenomedafoto.xlsx](https://github.com/unidoscontraacorrupcao/ej-server/files/2279737/exemplo_dados_candidatos_versaocomlinksenomedafoto.xlsx)

Observações:
1) não temos os dados de processos e bens ainda pois serão implementados no milestone 3.2 - por hora vamos importar sem
2) para as fotos usaremos a imagem padrão feita pela Luiza. assim que ela subir no asset vou renomear e juntar aqui também. Note que o padrão do nome da foto é o "\<cpf\>.jpg". Para fins dessa nossa primeira importação mantive apenas os 3 primeiros com o nome real, as outras a gente importa de um mesmo arquivo com o nome "99999999999.jpg"
3) como não temos os dados dos deputados ainda, repliquei alguns senadores apenas mudando algumas informações, pra podermos homologar essa exibição

Enfim, creio que já é possível concluir essa etapa dessa funcionalidade.

**Comment by ricardopoppi** [comment]
@davidCarlos segue imagens que devem ser utilizadas nessa importação. Nesse pacote, há as três cores, mas nesse momento devemos importar apenas a versão cinza: 
[drive-download-20180813T184008Z-001.zip](https://github.com/unidoscontraacorrupcao/ej-server/files/2284296/drive-download-20180813T184008Z-001.zip)

**Comment by parrahenri** [comment]
Virá direot do surveymonkey. coluna com texto contendo justificativa escrita pelos candidatos cadastrados.

Coluna na planilha: FALTANDO COLUNA NA PLANILHA (texto de máx 500 caracteres) ***@parrahenri sabe dizer como isso virá? A planilha com dados reais tem só um campo sobre novas medidas, onde virá o sem reposta/sim/não. Esse detalhamento virá em que campo?

**Comment by ricardopoppi** [comment]
@parrahenri ok! Planilha atualizada com esse novo campo que virá do survey.

@davidCarlos planilha e instruções atualizadas no comment acima: https://github.com/unidoscontraacorrupcao/ej-server/issues/99#issuecomment-414875441
